README.md

```diff
@@ -1,322 +1,371 @@
+             [Values]
+             Any numerical value (Default: 0.01)
+
+ --h_band    Maximum temporal filtering bandwidth in Hz
+             [Values]
+             Any numerical value (Default: 0.25)
+
+ --smooth    Spatial smoothing FWHM in mm, which determines the spatial smoothing sigma
+             [Values]
+             Any numerical value (Default: smfwhm=3 (mm))
+
+ --atlas     Name of the file to use as the EPI atlas
+             [Values]
+             Any string value with the relative path of the file (Default: ./lib/tmp/<model>EPIatlas.nii)
+```
+The above documentation can also be retrieved from the command line via `help` argument:
+
+    ./preproc_script_2.sh --help
+
+<a name="section-4-3-1"></a>
+#### 4.3.1 EPI registration estimation and wm/csf mask generation
+    a. Estimated by antsRegistration: rigid + affine + deformable (3 stages) transformation
+    b. Generate wm and/or csf mask: for rat brains, this is generated by the inverse transformation estimated in a.
+One can check the alignment of "EPI_n4_brain_regWarped.nii.gz" with the EPI template.\
+Output: \_n4_brain_reg
+
+<a name="section-4-3-2"></a>
+#### 4.3.2 Non-brain tissue noise estimation by PCA
+    a. Generate a mask for non-brain tissues
+    b. Extract the top 10 PCs from the masked tissues.
+    
+<a name="section-4-3-3"></a>
+#### 4.3.3 Nuisance regressions: 26 possible regressors (Chuang et al., 2018) & a user specified file of regressors
+    a. 3 for detrends: constant, linear, and quadratic trends
+    b. 10 PCs from non-brain tissues
+    c. 6 motion regressors (based on motion correction results) 
+    d. 6 motion derivative regressors: the temporal derivative of each motion regressor
+    e. csf or/and wmcsf signals 
+    f. one *.txt file containing user specified regressors (e.g., task patterns to be regressed)
+The script also generates a default outputs (0EPI_\*) which only regresses out the 3 trends (a). One can specify any combinations of above regressors in the command line. If you are preprocessing a group dataset, the same combination of regressors will be applied to all data.
+
+<a name="section-4-3-4"></a>
+#### 4.3.4 Normalization & temporal filtering
+    a. Normalize the regressed signals
+    b. Bandpass filter: bandwidth depends on the use of anesthesia
+    	e.g., 0.01–0.1Hz for iso and 0.01–0.25Hz for dmed (Pan et al., 2013)
+Output: \_mc_c_norm_fil
+
+<a name="section-4-3-5"></a>
+#### 4.3.5 EPI template registration & spatial smoothing & seed extraction
+    a. EPI template registration: transform cleaned-up data to template space by the transformation matrix estimated in (2.a)
+    b. Use Gaussian kernel for spatial smoothing. Set the FWHM value in mm in command line options
+    c. Extract the averaged timeseries based on atlas.
+Output: \_mc_c_norm_fil_reg_sm, \_mc_c_norm_fil_reg_sm_seed.txt
+
+The resulting FC map (FC.mat) for each data sample is included in the data folder.
+
+<a name="section-5"></a>
+## 5. References

```

Comment:
Was there anything added to the README section in this branch besides the references section? The file diff looks like the whole file was replaced.

Reply:
Yes, I completed rewrote the README.